build.sbt

```diff
@@ -1,12 +1,12 @@
 name := "api-integrations"
 
 ThisBuild / organization := "com.quantemplate"
-ThisBuild / scalaVersion := "3.0.0-RC2"
-ThisBuild / version := "0.1.3"
+ThisBuild / scalaVersion := "3.0.0"
+ThisBuild / version := "0.1.4"
 
-val AkkaVersion = "2.6.12"
+val AkkaVersion = "2.6.14"
 val AkkaHttpVersion = "10.2.4"
-val CirceVersion = "0.14.0-M3"
+val CirceVersion = "0.14.1"
 
 lazy val root = (project in file("."))
   .aggregate(
```

Comment:
I just realised that as long as we have 1 subproject this build file could be simplified.
It's just note. I don't expect changes.